authors/apps/profiles/tests/test_profile.py

```diff
@@ -0,0 +1,42 @@
+"""
+Test profiles app
+"""
+from django.urls import reverse
+
+from authors.apps.profiles.tests.test_base import BaseTestCase
+
+from .test_data import (new_user, update_profile)
```

Comment:
hey @wasibani-roy, it looks like you are not using this import ,you can always remove all unused comments to keep the code clean

Reply:
done